watchtower-plugin/src/net/http.rs

```diff
@@ -502,6 +522,75 @@ mod tests {
         assert!(matches!(error, AddAppointmentError::ApiError { .. }));
     }
 
+    #[tokio::test]
+    async fn test_request() {
```

Comment:
Test this method both with `POST` and `GET`

Reply:
This also applies for `test_request_connection_error`, do test it with the two allowed methods.

Reply:
sure!